ipc.rs

```diff
@@ -179,7 +180,7 @@ impl<T> IpcSender<T> where T: Serialize {
 
     pub fn to_opaque(self) -> OpaqueIpcSender {
         OpaqueIpcSender {
-            os_sender: self.os_sender,
+            os_sender: self.os_sender.clone(),
```

Comment:
Why do you need to clone here? Isn't self consumed anyway? Or am I missing something?...